### PR TITLE
add doc for slow-queries in dashboard

### PR DESCRIPTION
**Pull request by crazycs520**
### What is changed, added or deleted? (Required)

add doc for slow-queries in dashboard
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- No

### CODE REVIEW

**Comment by ran-huang** [comment]
1. If this is a new file, please add it to TOC and don't add an aliases.
2. Don't put the file in a deep directory; simply `/dashboard` would be enough.
3. The file name should be several words that contains key information, so users can understand the main information by looking at the URL. For example, `dashboard-slow-query-list.md`.

File: dashboard/slow-queries/list.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: 访问列表页面
```
Comment (scsldb) [inline dashboard/slow-queries/list.md:2]: 从内容来看是显示慢查询的页面，为什么取名访问列表页面呢？

File: dashboard/slow-queries/list.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+category: how-to
+aliases: ['/docs-cn/dev/how-to/???']
+---
+
+# 访问列表页面
+
+该功能可以查询整个集群中指定时间内执行时间较长的 SQL 查询。
```
Comment (scsldb) [inline dashboard/slow-queries/list.md:9]: 查询集群中所有慢查询语句，系统默认将执行时间超过 Session 变量 [tidb_slow_log_threshold](https://pingcap.com/docs-cn/stable/tidb-specific-system-variables/#tidb_slow_log_threshold) 或者 配置参数 [slow-threshold](https://pingcap.com/docs-cn/stable/tidb-configuration-file/#slow-threshold) 值的 SQL 语句视为慢查询语句并记录到系统[慢查询日志](https://pingcap.com/docs-cn/stable/identify-slow-queries/) 中。此功能通过内部 SQL 将查询结果输出到页面。

File: dashboard/slow-queries/list.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+默认情况下，执行时间超过 300ms 的 SQL 就会被视为慢查询，被记录到 [慢查询日志](https://pingcap.com/docs-cn/stable/identify-slow-queries/) 中，并可通过本功能对记录到的慢查询进行查询。可调整 [tidb_slow_log_threshold](https://pingcap.com/docs-cn/stable/tidb-specific-system-variables/#tidb_slow_log_threshold) SESSION 变量或 TiDB [slow-threshold](https://pingcap.com/docs-cn/stable/tidb-configuration-file/#slow-threshold) 参数调整慢查询阈值。
+
+> 注意：
+> 若关闭了慢查询日志，则本功能不可用。慢查询日志默认开启，可通过修改 TiDB 配置 [enable-slow-log](https://pingcap.com/docs-cn/stable/tidb-configuration-file/#enable-slow-log) 开启或禁用。
+
+## 访问列表页面
+
+可以通过以下两种方法访问集群诊断页面：
```
Comment (scsldb) [inline dashboard/slow-queries/list.md:18]: 集群诊断页面 是什么？

Comment (scsldb) [inline dashboard/slow-queries/list.md:18]: 访问页面的两种方式，如下：

File: dashboard/slow-queries/list.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+![access 访问页面](/media/dashboard/slow-queries/access.png)
+
+* 在浏览器中访问 [http://127.0.0.1:2379/dashboard/#/slow_query](http://127.0.0.1:2379/dashboard/#/slow_query)（将 127.0.0.1:2379 替换为任意实际 PD 地址和端口）。
+
+慢查询页面所展示的所有数据都来自于 TiDB 慢查询系统表及慢查询日志，参见 [慢查询日志](https://pingcap.com/docs-cn/stable/identify-slow-queries/) 文档了解详细情况。
+
+### 修改列表过滤条件
+
+页面顶部可修改显示的时间范围、按执行慢查询所在数据库进行过滤、按 SQL 关键字过滤、按 SQL 类型进行过滤及显示的数据条数，如下所示。默认显示 30 分钟内最近 100 条慢查询。
```
Comment (scsldb) [inline dashboard/slow-queries/list.md:30]: 可按时间范围、慢查询语句关联的数据库、SQL 关键字、SQL 类型、显示的慢查询语句数量等条件过滤，筛选慢查询句。如下所示，默认显示 30 分钟内最近 100 条慢查询。

File: dashboard/slow-queries/list.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+![图 1 ](/media/dashboard/slow-queries/list1.png)
+
+
+### 显示其他字段
```
Comment (scsldb) [inline dashboard/slow-queries/list.md:35]: 展开显示列信息

**Comment by ran-huang** [comment]
@crazycs520 Please address comments and fix CI. Thanks!

File: dashboard/slow-queries/list.md (hunk 5 above)
Comment (crazycs520) [reply dashboard/slow-queries/list.md:35]: 显示更多列信息？

File: dashboard/slow-queries/list.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+在列表中点击任意一行可以显示该慢查询的详细执行信息，包含：
+
+- SQL：慢查询 SQL 文本（下图中区域 1）
+- 执行计划：慢查询的执行计划，参阅「理解 TiDB 执行计划」文档了解如何解读执行计划（下图中区域 2）
+- 其他分类好的 SQL 执行信息（下图中区域 3）
+
+![detail 示例](/media/dashboard/slow-queries/detail1.png)
+
+点击「展开」（Expand）链接可以展开相应项的完整内容，点击「复制」（Copy）链接可以复制完整内容到剪贴板。
```
Comment (ran-huang) [inline dashboard/slow-queries/list.md:56]:
```suggestion
点击**展开**（**Expand**）链接可以展开相应项的完整内容，点击**复制**（**Copy**）链接可以复制完整内容到剪贴板。
```

File: dashboard/slow-queries/list.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+列表默认以**结束运行时间**（**Finish Time**）逆序排序，点击不同的列标题可以修改排序依据或切换排序顺序：
+
+![图 3 ](/media/dashboard/slow-queries/list3.png)
+
+## 查看执行详情
+
+在列表中点击任意一行可以显示该慢查询的详细执行信息，包含：
+
+- SQL：慢查询 SQL 文本（下图中区域 1）
+- 执行计划：慢查询的执行计划，参阅「理解 TiDB 执行计划」文档了解如何解读执行计划（下图中区域 2）
```
Comment (breeswish) [inline dashboard/slow-queries/list.md:51]:
```suggestion
- 执行计划：慢查询的执行计划，参阅[理解 TiDB 执行计划](https://pingcap.com/docs-cn/dev/query-execution-plan/)文档了解如何解读执行计划（下图中区域 2）
```

File: dashboard/slow-queries/list.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
+---
+title: 访问慢查询列表页面
+category: how-to
+---
+
+# 访问慢查询列表页面
+
+查询集群中所有慢查询语句，系统默认将执行时间超过 Session 变量 [tidb_slow_log_threshold](https://pingcap.com/docs-cn/stable/tidb-specific-system-variables/#tidb_slow_log_threshold) 或者配置参数 [slow-threshold](https://pingcap.com/docs-cn/stable/tidb-configuration-file/#slow-threshold) 值的 SQL 语句视为慢查询语句，系统会自己记录到[慢查询日志](https://pingcap.com/docs-cn/stable/identify-slow-queries/) 中。此功能通过 SQL 将查询结果输出到页面。
```
Comment (breeswish) [inline dashboard/slow-queries/list.md:8]: Do we need to use something like `/tidb-specific-system-variables.md#tidb_slow_log_threshold`? @lilin90

Comment (ran-huang) [reply dashboard/slow-queries/list.md:8]: @breeswish Yes. We need to use a relative path for this.

**Comment by ran-huang** [comment]
@crazycs520 Hi, I changed the file name and directory. PTAL if it's okay.

File: dashboard/slow-queries/list.md (hunk 5 above)
Comment (scsldb) [reply dashboard/slow-queries/list.md]: 原来叫显示字段信息，改成 显示列信息或者显示更多列信息

**Pull request merged**